buildlib/pr/io_demo/az-stage-io-demo.yaml

```diff
@@ -11,6 +11,10 @@ parameters:
   default: $(roce_iface)
 - name: initial_delay
   default: 20
+- name: server_num_paths
```

Comment:
now that we merged https://github.com/yosefe/ucx/pull/256, @changchengx can you please pass the different number of paths same as here : https://github.com/openucx/ucx/blob/440fb8333d077d826705e0b8b69301346f266e9e/buildlib/pr/io_demo/io-demo.yml#L78

It means that probably most of the AzureCI code in this PR should be removed, and just add 2 new tests to the list with different number of paths: server=1/client=2 and server=2/client=1

Reply:
@yosefe Done